.github/workflows/check-generated.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.22.2"
+          go-version: "1.22.3"
```

Comment:
Is it possible to define a global variable for the Go version and reference it throughout the code? This way, we won't have to update the Go version in multiple locations. Though, for a follow-up PR, not the current one.

Reply:
https://github.com/celestiaorg/celestia-core/issues/1349